app/src/test/kotlin/com/wire/android/ui/authentication/login/email/LoginEmailViewModelTest.kt

```diff
@@ -18,6 +18,8 @@
  *
  */
 
+@file:Suppress("MaxLineLength")
```

Comment:
I don't see why with these changes this suppress will be needed as they all seem to respect the max line length 🤷

Reply:
They don't tho :D